datasets/open_subtitles/open_subtitles.py

```diff
@@ -0,0 +1,122 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+This is a new collection of translated movie subtitles from http://www.opensubtitles.org/.
+
+IMPORTANT: If you use the OpenSubtitle corpus: Please, add a link to http://www.opensubtitles.org/ to your website and to your reports and publications produced with the data!
+
+This is a slightly cleaner version of the subtitle collection using improved sentence alignment and better language checking.
+
+62 languages, 1,782 bitexts
+total number of files: 3,735,070
+total number of tokens: 22.10G
+total number of sentence fragments: 3.35G
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/OpenSubtitles.php"
+_CITATION = """\
+P. Lison and J. Tiedemann, 2016, OpenSubtitles2016: Extracting Large Parallel Corpora from Movie and TV Subtitles. In Proceedings of the 10th International Conference on Language Resources and Evaluation (LREC 2016)
```

Comment:
Can you use te bibtex format here actually ?
(I just noticed, sorry)

Feel free to create a new PR